src/ButteredBread.sol

```diff
@@ -103,28 +90,11 @@ contract ButteredBread is ERC20VotesUpgradeable, OwnableUpgradeable, IButteredBr
      * @notice Set LP token scaling factor
      * @param _lp Liquidity Pool token
      * @param _factor Scaling percentage incentive of LP token (e.g. 100 = 1X, 150 = 1.5X, 1000 = 10X)
-     * @param _sync Automatically sync all accounts voting weights with new factor
-     * Note: In case of DDOS, set _sync to false and manually sync with syncVotingWeights(_depositors(), _lp)
      */
-    function modifyScalingFactor(address _lp, uint256 _factor, bool _sync) external onlyOwner {
-        _modifyScalingFactor(_lp, _factor);
-        if (_sync) {
-            for (uint256 i = 0; i < _depositors.length; i++) {
-                _syncVotingWeight(_depositors[i], _lp);
-            }
-        }
+    function modifyScalingFactor(address _lp, uint256 _factor, address[] calldata holders) external onlyOwner {
```

Comment:
`_factor` shouldn't be an argument, it should pull from `scalingFactors` based on `_lp` arugment